src/modules/portfolio/selectors/select-markets-open-orders.js

```diff
@@ -1,19 +1,31 @@
 import { createSelector } from "reselect";
 import store from "src/store";
-import getOpenOrders from "modules/orders/selectors/open-orders";
 import * as constants from "src/modules/common-elements/constants";
+import { selectMarkets } from "modules/markets/selectors/markets-all";
+import getUserFilledOrders from "modules/orders/selectors/filled-orders";
+import getUserOpenOrders from "modules/orders/selectors/user-open-orders";
 
 export default function() {
   return marketsOpenOrders(store.getState());
 }
 
-export const marketsOpenOrders = createSelector(getOpenOrders, openOrders => {
-  const markets = openOrders.filter(
-    market => market.marketStatus !== constants.MARKET_CLOSED
-  );
+export const marketsOpenOrders = createSelector(selectMarkets, allMarkets => {
+  const markets = allMarkets.reduce((p, m) => {
+    if (m.marketStatus === constants.MARKET_CLOSED) return p;
+    const userOpenOrders = getUserOpenOrders(m.id);
+    if (userOpenOrders.length === 0) return p;
```

Comment:
<img width="542" alt="Screen Shot 2019-04-26 at 2 32 30 PM" src="https://user-images.githubusercontent.com/6775839/56837690-24679d80-6830-11e9-973c-c74925dd664a.png">
got this error here